CHANGES.md

```diff
@@ -1,5 +1,35 @@
 # Changes
 
+## edge-21.9.1
+
+This release includes various improvements and feature additions across the policy
+feature i.e, New validating webhook for policy resources. This also includes changes
+in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
+in the proxy authorization metrics. In addition, proxy injector has also been updated
+to set the right `opaque-ports` annotation on services with default opaque ports.
+
+* Added a new validating admission controller to validate the proxy resources
+* Updated the proxy-init to remove a rule which caused the packets from the proxy
+  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
+* Added new `LINKERD_DOCKER_REGISTRY` env variable to configure the docker
+  registry in the CLI
+* Updated inbound policy enforcement to interrupt TCP forwarding if a previously
+  established authorization is revoked
+* Added new proxy metrics w.r.t Authorization along with traffic target labels
+* Updated inbound TCP metrics to only include a `srv_name` label
+* Updated the proxy to export route-oriented metrics only when a ServiceProfile
+  is enabled.
+* Updated tokio to include performance improvements and to enable link-time
```

Comment:
nit: do you think we should exclude this? doesn't seem very user facing.

Reply:
Not very user facing but perf improvements felt important to be mentioned! 🤔

Reply:
Hmm, We can remove the link-time part I guess. Updating

Reply:
I'm good either way, I see your point too. Just asked to see what you think :)